### DISCUSSION TITLE
How can I customize the title block?

### DISCUSSION THREAD

**Discussion by fredguth** [quarto-dev/quarto-cli#9427] [Q&A] [answered]
### Description

I am developing a `project: website` and I want to enhance the title block on my project pages to include not just the main title but also a subtitle and a description. Are there specific variables available for adding a subtitle and description in Quarto? If customization is required, how can I modify the title block html? Would I need to create a custom partial? If so, could you provide an HTML example of how such a partial might look?

**Comment by mcanouil** [comment, marked as answer]
See <https://quarto.org/docs/journals/templates.html#html-partials>.
Look at the source code provided.
It's HTML, nothing more.
